pandas/core/frame.py

```diff
@@ -1703,6 +1703,7 @@ def to_html(self, buf=None, columns=None, col_space=None, header=True,
             Character recognized as decimal separator, e.g. ',' in Europe
 
             .. versionadded:: 0.18.0
+
```

Comment:
Is this one actually needed? (the blank line) 
I always assumed it was not needed (the one before *is* needed) 
(but note I am not against this change, because without it looks a bit strange in the plain text version)

Reply:
Again, not sure if the newline was necessary. There were several "unexpected end to block quote" warnings. I can check my logs.

Reply:
don't mind this one. As I said, I think the change is also good from "aesthetic" point of view, whether it was actually needed or not to remove a warning